clients/da-vinci-client/src/main/java/com/linkedin/davinci/kafka/consumer/StoreIngestionTask.java

```diff
@@ -168,7 +168,7 @@ public abstract class StoreIngestionTask implements Runnable, Closeable {
   protected final String kafkaVersionTopic;
   protected final String realTimeTopic;
   protected final String storeName;
-  private final boolean isUserSystemStore;
+  protected final boolean isUserSystemStore;
```

Comment:
Do we need this change? There is an existing public method `isUserSystemStore()`. Could we use this method in `canSwitchToLeaderTopic`?

Reply:
Thanks Shuhui. Fixed it.